docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -0,0 +1,300 @@
+the user to avoid overloading the {fleet-server} from artifact API calls.
+
+`artifact_limit.interval`:::
+How often artifacts are rolled out. Default of 100ms allows 10 artifacts to be
+rolled out per second.
+
+`artifact_limit.burst`:::
+Number of transactions allowed for a burst, controlling oversubscription on
+outbound buffer.
+
+`ack_limit.max`:::
+Maximum number of agents that can call the Ack API concurrently. It allows the
+user to avoid overloading the {fleet-server} from Ack API calls.
+
+`ack_limit.interval`:::
+How often an acknowledgment (ACK) is sent. Default value of 10ms enables 100
+ACKs per second to be sent.
+
+`ack_limit.burst`:::
+Burst of ACKs to accommodate (default of 20) before falling back to the rate
+defined in `interval`.
+
+`enroll_limit.max`:::
+Maximum number of agents that can call the Enroll API concurrently. This setting
+allows the user to avoid overloading the {fleet-server} from Enrollment API
+calls.
+
+`enroll_limit.interval`:::
+Interval between processing enrollment request. Enrollment is both CPU and RAM
+intensive, so the number of enrollment requests needs to be limited for overall
+system health. Default value of 100ms allows 10 enrollments per second.
+
+`enroll_limit.burst`:::
+Burst of enrollments to accept before falling back to the rate defined by
+`interval`.
+
+[discrete]
+[[scaling-recommendations]]
+=== Scaling recommendations
+
+The following tables provide resource requirements and scaling guidelines based
+on the number of agents required by your deployment:
+
+* <<resource-requirements-by-number-agents>>
+* <<recommend-settings-scaling-agents-a>>
+* <<recommend-settings-scaling-agents-b>>
+
+// Nima: Are these guidelines all cloud-specific? What about on prem/self-managed?
+
+// Note: I don't like how the spanned rows under vCPU look. Do you think it's OK
+// or should I just repeat the values?
+
+// Nima: I'm looking at  elastic cloud, and the drop-down list of options that
+// I see does not match these settings. Does this table need to be updated to
+// show the new values?
+
+[discrete]
+[[resource-requirements-by-number-agents]]
+==== Resource requirements by number of agents
+|===
+| Number of Agents | Memory      | vCPU           | {es} Cluster size
+
+| 50               | 512MB    .5+| Up to 2.5 vCPU | 480GB disk \| 16GB RAM \| up to 5 vCPU
+| 5,000            | 1GB                          | 960GB disk \| 32GB RAM \| 5 vCPU
+| 7,500            | 2GB                          | 1.88TB disk \| 64GB RAM \| 9.8 vCPU
+| 10,000           | 4GB                          | 3.75TB disk \| 128GB RAM \| 19.8 vCPU
+| 12,500           | 8G                           | 7.5TB disk \| 256GB RAM \| 39.4 vCPU
+| 30,000           | 16GB     .2+| 2.5 vCPU       | 7.5TB disk \| 256GB RAM \| 39.4 vCPU
+| 50,000           | 32GB                         | 11.25TB disk \| 384GB RAM \|59.2 vCPU
+|===
+
+// Nima: I tried multiple iterations of this table (and the next) and none were
```

Comment:
what if we add it as a .png for now?
I am working on getting some development done which should move away from these variables

Reply:
I don't think that's a good solution because it's not accessible and probably won't be any more readable than a wide table. If you want all the content to be all in one table, I can do that, and users will need to scroll right.